lib/helpers/createNewChannelReport.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
+
+const log              = require('../../getLogger')('channel report');
+const reportForChannel = require('./reports/forChannel');
+
+module.exports = function createNewChannelReport(bot, channel, standups) {
+  let report = reportForChannel(channel, standups);
+
+  log.verbose('Sending report for ' + channel.name);
+
+  bot.say(report, function(err, response) {
+    if (err) {
+      console.log(err);
```

Comment:
Ideally I would want the log object to be used, and also the ability to mock the log object without having to do funky stuff with require. That would involve dependency injecting the logger in some way. Might be attached to a grab bag of data that gets passed around to these kinds of function or ???. Not sure don't know the whole app well enough.